skyportal/handlers/api/news_feed.py

```diff
@@ -61,13 +61,41 @@ def fetch_newest(model):
                 .all()
             )
 
+        def fetch_newest_comments():
+            owned_comments = (
+                Comment.query.filter(
+                    Comment.obj_id.in_(
+                        DBSession()
+                        .query(Source.obj_id)
+                        .filter(
+                            Source.group_id.in_(
+                                [g.id for g in self.current_user.accessible_groups]
+                            )
+                        )
+                    )
+                )
+                .order_by(desc(Comment.created_at or Comment.saved_at))
```

Comment:
I don't think `Comment` has a `saved_at` column -- this just applies to the more general function above (`Source` has a `saved_at` column)
```suggestion
                .order_by(desc(Comment.created_at))
```

Reply:
n/a if this function is removed entirely as suggested below